frontend/express/public/javascripts/countly/countly.views.js

```diff
@@ -3782,183 +3782,403 @@ window.DownloadView = countlyView.extend({
 
 window.LongTaskView = countlyView.extend({
 	initialize:function () {
-		this.template = Handlebars.compile($("#table-template").html());
-    },
-    beforeRender: function() {
-        return $.when(countlyTaskManager.initialize()).then(function () {});
-    },
-    getStatusColor: function(status){
-      if(status === "completed")
-          return "#2FA732";
-      if(status === "errored")
-          return "#D63E40";
-      return "#E98010";
-    },
-    renderCommon:function (isRefresh) {
-        var types = {
+        this.template = Handlebars.compile($("#report-manager-template").html());
+        this.taskCreatedBy = 'manually';
+        this.types = {
             "all": jQuery.i18n.map["common.all"],
             "funnels": jQuery.i18n.map["sidebar.funnels"] || "Funnels",
             "drill": jQuery.i18n.map["drill.drill"] || "Drill"
         };
-        var states = {
+         
+        this.runTimeTypes = {
+            "all": jQuery.i18n.map["common.all"],
+            "auto-refresh": jQuery.i18n.map["taskmanager.auto"],
+            "none-auto-refresh": jQuery.i18n.map["taskmanager.manual"] 
+        }
+
+        this.states = {
             "all": jQuery.i18n.map["common.all"],
             "running":jQuery.i18n.map["taskmanager.running"],
             "rerunning":jQuery.i18n.map["taskmanager.rerunning"],
             "completed":jQuery.i18n.map["taskmanager.completed"],
             "errored":jQuery.i18n.map["taskmanager.errored"]
         };
+    },
+    beforeRender: function() {
+        return $.when(countlyTaskManager.initialize(null, {"report_desc": {$ne:""}})).then(function () {});
```

Comment:
Do we intend to initialize task data only with tasks that have descriptions?

Reply:
Now with descriptions are recognized as  manual created

Reply:
So per this code, on view render, in the table we will list only tasks where description is not empty. why?